src/routes/(app)/select-namespace/+page.svelte

```diff
@@ -64,42 +59,28 @@
       bind:value={searchValue}
     />
   </form>
-  <div class="h-screen w-full" aria-label={translate('common.namespaces')}>
-    {#if namespaceList.length}
-      {#if filteredList.length}
-        <VirtualList items={filteredList} let:item itemHeight={50}>
-          {@const first = item === filteredList[0]}
-          {@const last = item === filteredList[filteredList.length - 1]}
-          <div class="link-item" class:first class:last>
-            <button
-              class="w-full truncate p-3 text-left"
-              on:click={() => item?.onClick(item.namespace)}
-            >
-              {item.namespace}
-            </button>
-          </div>
-        </VirtualList>
-      {:else}
-        <EmptyState title={translate('common.no-results')} />
-      {/if}
+  <div
+    class="group h-screen w-full"
```

Comment:
This list actually goes off screen with `h-screen`. Wondering if we want to add a `calc()` here to set the max height so it does appear all on screen.

Reply:
This is why we never used the virtual list on the event history. I struggled with this for a long time in 2021. 🤣 I might play around with it tomorrow to see what I can do.